harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/HarvesterJob.java

```diff
@@ -79,16 +91,16 @@ public void setHarvester(AbstractHarvester<?> harvester) {
     @Override
     public void interrupt() throws UnableToInterruptJobException {
         harvester.cancelMonitor.set(true);
-        
+
         // Following the suggestion of InterruptableJob
         // Sometimes the harvester is frozen
         // give some time, but if it does not finish properly...
         // just kill it!!
-         new Thread(){
+        new Thread() {
             @Override
             public void run() {
                 super.run();
-                
```

Comment:
Try to leave whitespaces unchanged on PR.

Reply:
I always try to format the files I'm changing. Make little changes that doesn't cost you much every time you can to improve the overall code quality: refactor variables, methods, rename to more legible name, formatting the file, ...